vmdb/spec/controllers/ops_controller/diagnostics_spec.rb

```diff
@@ -126,5 +126,17 @@
         controller.send(:log_depot_edit)
       end
     end
+
+    context "#tree_select" do
+      it "renders zone list for root node" do
+        controller.instance_variable_set(:@_response, ActionController::TestResponse.new)
+        controller.instance_variable_set(:@sb, :active_tree => :diagnostics_tree)
+        controller.instance_variable_set(:@layout, 'ops')
+        controller.instance_variable_set(:@_params, :id => 'root', :action => 'tree_select')
+        controller.send(:tree_select)
```

Comment:
If there is, I'd like to learn about it. I was literally copying parts of other tests to get this working. Can you e.g. point me to a test (that we have, or is elsewhere), that would be doing it right?